InvenTree/report/api.py

```diff
@@ -683,13 +684,36 @@ def get(self, request, *args, **kwargs):
         return self.print(request, orders)
 
 
+class ReportMetadata(RetrieveUpdateAPI):
+    """API endpoint for viewing / updating Report metadata."""
+
+    def get_serializer(self, *args, **kwargs):
+        """Return correct MetadataSerializer instance depending on which model is requested"""
+        # Get type of report, make sure its one of the allowed values
+        reporttype = self.kwargs.get('reporttype', 'PurchaseOrder')
```

Comment:
Question: Is it intended that no argument = PO? Is there a need for a default value or could we raise it if no kwarg is defined? To make sure it is better debuggable.